images/scripts/foonux.bootstrap

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# mock image which doesn't need any qemu/kvm; used by cockpituous integration tests
+echo fakeimage > $1
```

Comment:
I feel like it might be interesting to inject a timestamp here so we get a different hash each time.

Reply:
Good idea!